### PR TITLE
Dropdown fixes

### PR DESCRIPTION
**Pull request by muhtasimhafiz**
#### Description

Please include a short description of the suggested change and the reasoning behind the approach you have chosen.

#### Link to ticket

Please add a link to the GitHub issue being addressed by this change.

### Type

[] Fix
[x] Feature
[] Cleanup 

#### Screenshot of the result

If your change affects the user interface, you should include a screenshot of the result with the pull request.

### CODE REVIEW

File: app/Domain/Tickets/Templates/showAll.blade.php [hunk 1]
```diff
@@ -126,261 +126,238 @@
...
                             </td>
 
-                        <td data-order="<?=$tpl->e($row['headline']); ?>">
-                            <?php if ($row['dependingTicketId'] > 0) { ?>
-                                <small><a href="#/tickets/showTicket/<?=$row['dependingTicketId'] ?>"><?=$tpl->escape($row['parentHeadline']) ?></a></small> //<br />
-                            <?php } ?>
-                            <a href="#/tickets/showTicket/<?=$tpl->e($row['id']); ?>"><?=$tpl->e($row['headline']); ?></a></td>
+                            <td data-order="<?= $tpl->e($row['headline']) ?>">
+                                <?php if ($row['dependingTicketId'] > 0) { ?>
+                                <small><a
```
Comment (marcelfolaron) [inline app/Domain/Tickets/Templates/showAll.blade.php:131]: Is this the code formatting tool again? Why was that changed?

File: app/Domain/Tickets/Templates/showAll.blade.php [hunk 2]
```diff
@@ -126,261 +126,238 @@
...
-                                    <!-- Menu Header -->
-                                    <li class="nav-header border">{{ __('dropdown.how_big_todo') }}</li>
-
-                                    <!-- Dynamic Effort Options -->
-                                    @foreach ($efforts as $effortKey => $effortValue)
-                                        <x-global::actions.dropdown.item href="javascript:void(0);"
-                                            data-value="{{ $row['id'] . '_' . $effortKey }}"
-                                            id="ticketEffortChange{{ $row['id'] . $effortKey }}">
-                                            {{ $effortValue }}
+                                <div class="dropdown ticketDropdown effortDropdown show">
```
Comment (marcelfolaron) [inline app/Domain/Tickets/Templates/showAll.blade.php:221]: Why this line here? That is part of the dropdownPill component

File: app/Domain/Tickets/Templates/showAll.blade.php [hunk 3]
```diff
@@ -126,261 +126,238 @@
...
-                                    </x-global::content.context-menu>
 
-                                </div>
-                            </td>
                             <?php
-
-                            if ($row['sprint'] != '' && $row['sprint'] != 0 && $row['sprint'] != -1) {
-                                $sprintHeadline = $tpl->escape($row['sprintName']);
+                            // Determine the user text and image to display
+                            if ($row['editorFirstname'] != '') {
```
Comment (marcelfolaron) [inline app/Domain/Tickets/Templates/showAll.blade.php:281]: Why is this up here when it could just be inside of <x-slot name="buttonText"> tag?

File: app/Domain/Tickets/Templates/showAll.blade.php [hunk 4]
```diff
@@ -126,261 +126,238 @@
...
+                                    <x-global::actions.dropdown.item class="nav-header border">
+                                        {{ __('dropdown.choose_list') }}
+                                    </x-global::actions.dropdown.item>
 
-                                    <!-- Not Assigned Option -->
-                                    <x-global::actions.dropdown.item href="javascript:void(0);" :data-label="__('label.not_assigned_to_list')"
+                                    <x-global::actions.dropdown.item href="javascript:void(0);" :data-label="$tpl->__('label.not_assigned_to_list')"
                                         :data-value="$row['id'] . '_0'">
-                                        {{ __('label.not_assigned_to_list') }}
+                                        {{ $tpl->__('label.not_assigned_to_list') }}
```
Comment (marcelfolaron) [inline app/Domain/Tickets/Templates/showAll.blade.php:338]: That $tpl-> is not needed

File: app/Domain/Tickets/Templates/showAll.blade.php [hunk 5]
```diff
@@ -394,8 +371,9 @@ class="priority-bg-{{ $priorityKey }}"
                                 $date = $date->format($tpl->__('language.dateformat'));
                             }
                             ?>
-                            <td data-order="<?=$row['dateToFinish'] ?>" >
-                                <input type="text" title="{{ __("label.due") }}" value="<?php echo $date ?>" class="quickDueDates secretInput" data-id="<?php echo $row['id'];?>" name="date" />
+                            <td data-order="<?= $row['dateToFinish'] ?>">
```
Comment (marcelfolaron) [inline app/Domain/Tickets/Templates/showAll.blade.php:374]: code formatter ?

File: app/Domain/Ideas/Templates/advancedBoards.blade.php [hunk 6]
```diff
@@ -92,24 +83,14 @@ class="delete">
...
-                                    <!-- Idea Kanban Link (Active) -->
-                                    <x-global::actions.dropdown.item href="{{ BASE_URL }}/ideas/advancedBoards"
-                                        class="active">
-                                        {!! __('buttons.idea_kanban') !!}
-                                    </x-global::actions.dropdown.item>
-                                </x-slot:menu>
-
-                            </x-global::actions.dropdown>
-
+                            <button class="btn btn-default dropdown-toggle"
```
Comment (marcelfolaron) [inline app/Domain/Ideas/Templates/advancedBoards.blade.php:86]: Why was this reverted? Seems it should be using the dropdown component?

File: app/Domain/Ideas/Templates/advancedBoards.blade.php [hunk 7]
```diff
@@ -24,48 +24,39 @@
             <h5><?php $tpl->e(session('currentProjectClient') . ' // ' . session('currentProjectName')); ?></h5>
             <h5>{{ session('currentProjectClient') . ' // ' . session('currentProjectName') }}</h5>
             @if (count($allCanvas) > 0)
-                <x-global::content.context-menu label-text="<i class='fa-solid fa-ellipsis-v'></i>" contentRole="link"
-                    position="bottom" align="start" class="headerEditDropdown btn btn-transparent">
-
-                    <x-slot:menu>
+                <span class="dropdown dropdownWrapper headerEditDropdown">
```
Comment (marcelfolaron) [inline app/Domain/Ideas/Templates/advancedBoards.blade.php:27]: Why was this reverted? I assume this is the 3 dot menu on the top right hand side? That should use the dropdown component.

File: app/Domain/Ideas/Templates/advancedBoards.blade.php [hunk 8]
```diff
@@ -136,32 +117,32 @@ class="editHeadline"><i class="fas fa-edit"></i></a>
...
-                                                            <x-global::content.context-menu
-                                                                label-text="<i class='fa fa-ellipsis-v' aria-hidden='true'></i>"
-                                                                contentRole="link" position="bottom" align="start"
-                                                                class="inlineDropDownContainer ticketDropDown"
-                                                                style="float:right;">
-
-                                                                <x-slot:menu>
-                                                                    <!-- Edit Item Header -->
-                                                                    <li class="nav-header">{!! __('subtitles.edit') !!}</li>
+                                                            <div class="inlineDropDownContainer" style="float:right;">
```
Comment (marcelfolaron) [inline app/Domain/Ideas/Templates/advancedBoards.blade.php:120]: Isn't that the card dropdown? Which should be a dropdown/context menu?

File: app/Domain/Ideas/Templates/advancedBoards.blade.php [hunk 9]
```diff
@@ -326,8 +307,7 @@ class="dropdown ticketDropdown userDropdown noBg show right lastDropdown dropRig
                     @endforeach
                 ];
                 leantime.ideasController.initIdeaKanban(ideaStatusList);
-                leantime.canvasController.initUserDropdown('ideas');
-
+                leantime.ideasController.initUserDropdown();
```
Comment (marcelfolaron) [inline app/Domain/Ideas/Templates/advancedBoards.blade.php:310]: The function was changed and it needs to be `leantime.canvasController.initUserDropdown('ideas');`

**Review by marcelfolaron — CHANGES_REQUESTED** [review]

File: app/Domain/Ideas/Templates/advancedBoards.blade.php (hunk 7 above)
Comment (muhtasimhafiz) [reply app/Domain/Ideas/Templates/advancedBoards.blade.php:27]: i was having issues in this fiel so I reverted it, and started replacing it piece by piece again

File: app/Domain/Tickets/Templates/showAll.blade.php (hunk 2 above)
Comment (muhtasimhafiz) [reply app/Domain/Tickets/Templates/showAll.blade.php]: removed this, thought this was necessary

File: app/Domain/Tickets/Templates/showAll.blade.php (hunk 1 above)
Comment (muhtasimhafiz) [reply app/Domain/Tickets/Templates/showAll.blade.php:131]: most likely

File: app/Domain/Tickets/Templates/showAll.blade.php (hunk 3 above)
Comment (muhtasimhafiz) [reply app/Domain/Tickets/Templates/showAll.blade.php:279]: bit confused about this one? what tag are you expecting?

**Review by marcelfolaron — APPROVED** [review]
